### PR TITLE
Add cautionary contact to raise a repair page

### PR DESCRIPTION
**Pull request by cdccollins**

### CODE REVIEW

File: app/views/repair_requests/new.html.erb [hunk 1]
```diff
@@ -10,6 +10,25 @@
           New repair for <%= @property.address %>
         </h1>
 
+        <div class="hackney-warning-labels">
+          <% if @cautionary_contacts.present? %>
+          <table class="hackney-cautionary-contact-table" id="hackney-cautionary-contact-raise-a-repair">
```
Comment (cdccollins) [inline app/views/repair_requests/new.html.erb]: I wasn't sure of the best way to add only this margin to the table on the raise a repair page, so I went with id.

**Comment by cdccollins** [comment]
Should I test whether it shows up on the raise a repair page?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -156,7 +156,6 @@ $govuk-breakpoints: (
     }
   }
 }
-
```
Comment (pudiva) [inline app/assets/stylesheets/application.scss]: may not be a big deal, but take changes that do nothing, they add noise to the history and may confuse us if we try to search it.

File: app/views/repair_requests/new.html.erb (hunk 1 above)
Comment (pudiva) [reply app/views/repair_requests/new.html.erb]: there is a govuk class for this:
https://design-system.service.gov.uk/styles/spacing/

Comment (pudiva) [reply app/views/repair_requests/new.html.erb]: also, it looks like this whole section is repeated on `properties#show`, isn't it? You should make a partial if that's the case.

**Review by pudiva — CHANGES_REQUESTED** [review]
almost there

**Comment by pudiva** [comment]
> Should I test whether it shows up on the raise a repair page?

Definitely. You can create a review app on Heroku if you need to show @Lawrencerichards.

File: app/views/repair_requests/new.html.erb (hunk 1 above)
Comment (cdccollins) [reply app/views/repair_requests/new.html.erb]: > https://design-system.service.gov.uk/styles/spacing/

Omg this is magic!

Comment (cdccollins) [reply app/views/repair_requests/new.html.erb]: > also, it looks like this whole section is repeated on `properties#show`, isn't it? You should make a partial if that's the case.

How would the varied margin work with a partial?

**Review by pudiva — APPROVED** [review]
:shipit:

**Pull request merged**